config/core/test-configs.yaml

```diff
@@ -339,8 +339,9 @@ test_plans:
       grp_test: "TMR"
       job_timeout: '30'
 
-  ltp-timers-qemu:
+  ltp-timers_qemu:
```

Comment:
In the test plan is still ltp-timers-qemu, we also need to change it.

Reply:
Sure.  Actually I think this fix can be squashed with the first commit.

Reply:
Done.